client/src/Utils/geoLocator.js

```diff
@@ -0,0 +1,9 @@
+const successCallback = position => {
+  console.log(position);
+};
+
+const errorCallback = error => {
+  console.error(error);
+};
+
+navigator.geolocation.getCurrentPosition(successCallback, errorCallback);
```

Comment:
don't see any `export` in the file, but it actually imported in `App` component. just interested - does it work?

Reply:
> Why does it work like that, I have not figured out exactly.

please make your code more predictable or find out why it works that way and can be used here :)

Reply:
Reworked the geoLocator.js file, now it activates only when called and it is being called in the API.js file, in order to supply current coordinates of the user to the app.